connectors-home/modules/ROOT/pages/introduction/connectors-shared-responsibilities.adoc

```diff
@@ -0,0 +1,90 @@
+
+= Anypoint Connectors Shared Responsibilities
+
+The successful operation of Anypoint Connectors is a shared responsibility between MuleSoft and the customer. It is critical to understand which areas customers are responsible for and which areas MuleSoft is responsible for.
+
+== Anypoint Connectors Categories
+
+Anypoint Connectors fall into these categories:
+
+* Connectors that enable the use of cross-platform protocols or open APIs. +
+For example, Database Connector, which is based on Oracle’s JDBC protocol.
+
+* Connectors or modules that enable connectivity to MuleSoft or Salesforce applications. +
+For example, Salesforce Marketing Cloud Connector and Anypoint MQ Connector.
+
+* Connectors that enable connectivity to third-party or partner APIs. +
+For example, Azure Key Vault Connector and Stripe Connector.
+
+* MuleSoft Certified Connectors +
+These are developed by MuleSoft’s partners and subsequently reviewed and certified by MuleSoft, for example ESRI Connector (in Anypoint Exchange identify them as MuleSoft Partner).
+
+* Community Connectors +
+These are developed by the developer community and subsequently reviewed and certified by MuleSoft, for example, Slack Connector - Mule 3 (in Anypoint Exchange identify them as Open-Source Community).
```

Comment:
Use the same formatting as above. I also have the same question about what "(in Anypoint Exchange identify them as Open-Source Community)" means.